modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -186,6 +190,31 @@ Every processed record of the Batch Job Instance starts with the same variables
 In the On Complete phase, none of these variables (not event the original ones) are visible. Only the final result is available in this phase. Moreover, since the Batch Job Instance executes asynchronously from the rest of the flow, no variable set in either a Batch Step or the On Complete phase will be visible outside the Batch Scope.
 
 
+=== Scheduling Strategy
+
+The Batch Job scope provides Scheduling Strategy options for controlling the
+execution of job instances.
+
+* ORDERED_SQUENTIAL (the default): If several job instances are in an executable
+state at the same time, the instances execute one at a time based on their
+creation timestamp.
++
+Note that there is no guarantee that the records will be executed in order.
```

Comment:
This doesn't belong here. This is true for all scheduling strategies.

Reply:
moved